draft-ietf-tls-dtls-connection-id.md

```diff
@@ -272,7 +272,8 @@ cid
    is in use.
 
 enc_content
-:  The encrypted form of the serialized DTLSInnerPlaintext structure.
+:  The AEAD-encrypted form of the serialized DTLSInnerPlaintext structure of 
+   length ciphertext_length. 
```

Comment:
Why is this just for AEAD cipher types?

Reply:
@jsalowey 

FMPOV, there are currently two interpretations of that definition. 

Hannes:
> Using DTLSCIDCiphertext.length is IMHO incorrect because the length of the encrypted content is different from the length of the entire DTLSCIDCiphertext structure. Do I see that wrong?

So xxx.length is the length of xxx.

My understanding:
> DTLSCIDCiphertext.length is not the length of DTLSCIDCiphertext, it's the value of the field uint16 length of that structure.

So xxx.length is the value of the field xxx.length.

I think, my interpretation is right.

If I'm right, then the changes for "easier readability" will change the term (field name) `length` in more places, not only there.

```sh
   ...
    uint16 ciphertext_length;
   opaque enc_content[DTLSCIDCiphertext.ciphertext_length];
} DTLSCIDCiphertext;

```

with that, also the term `length`in the text, not only in the structure, must be exchanged with `ciphertext_length`. And, of course, for the other cipher types as well.

With that, I'm not sure, if exchange that term really pays off, this style/expression is used in many other RFCs, and so should be common to the most readers.

(I guess, if @hannestschofenig finds the time to revisit this, he will agree on my interpretation and with that, it may either "change much more", or this PR may get withdrawn.)